src/toast/button/__snapshots__/button.spec.js.snap

```diff
@@ -1,48 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`common/Button graphical (JSX) should add custom className 1`] = `
-<button
-  className="button custom"
-  onClick={undefined}
-  style={Object {}}
->
-  
-</button>
-`;
+exports[`common/Button graphical (JSX) should add custom className 1`] = `null`;
```

Comment:
You should give text so it prints jsx :)

Reply:
oups...